Explorer/Assets/DCL/Interaction/Systems/WritePointerEventResultsSystem.cs

```diff
@@ -37,6 +37,8 @@ public partial class WritePointerEventResultsSystem : BaseUnityLoopSystem
 
         private readonly IComponentPool<RaycastHit> raycastHitPool;
 
+        private bool messageSent;
```

Comment:
Just to say that: it's not necessary to introduce a field in the class: you could pass it as `[Data] ref bool messageSent` from the local var